src/Chainweb/SPV/CreateProof.hs

```diff
@@ -428,7 +428,7 @@ minimumTrgHeader cutDb tcid scid bh = do
     trgHeader <- seekAncestor trgChain trgHeadHeader trgHeight >>= \case
         Just x -> return $! x
         Nothing -> throwM $ SpvExceptionTargetNotReachable
-            { _spvExceptionMsg = "target chain not reachabe. Chainweb instance is to young"
+            { _spvExceptionMsg = "target chain not reachabe. Chainweb instance is too young"
```

Comment:
Oops, there's another typo on this line.

Reply:
Huh, didn't spot that one :smile: